### PR TITLE
150 changes partial

### PR DESCRIPTION
**Pull request by rebeccaestes**
Added changes tab and essay field.

### CODE REVIEW

File: app/views/pregnancies/_menu.html.erb [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <ul class="nav nav-pills nav-stacked">
     <li class='active'><%= link_to 'Abortion Information', '#abortion_section', data: { toggle: 'tab' } %></li>
     <li><%= link_to 'Patient Information', '#patient_section', data: { toggle: 'tab' } %></li>
+    <li><%= link_to 'Changes', '#change_section', data: { toggle: 'tab' } %></li>
```
Comment (colinxfleming) [inline app/views/pregnancies/_menu.html.erb]: Thanks so much for sticking with this and hashing it out!

**Comment by colinxfleming** [comment]
Nice work, @rebeccaestes ! I suspect that the glyphicon stuff still being in your PR is causing a merge conflict, but other than that this is great. If you've got time, would be killer if you could iron out the merge conflict, but NBD if not.

**Comment by rebeccaestes** [comment]
Yeah, that was it - it should be fixed now. And thanks!

**Comment by colinxfleming** [comment]
Made a small tweak in #168 . Thanks a ton for your work on this @rebeccaestes and welcome aboard!

**Pull request closed without merging**